src/App.js

```diff
@@ -7,23 +7,20 @@ class App extends Component {
     super(props)
     this.state = {
       squares: [null, null, null, null, null, null, null, null, null],
-      turn: 0,
-      lastTurn: null
+      player: 1,
     }
   }
 
  markSquare = (index) => {
-  let {squares, turn} = this.state
-  let playerOne = "❌"
-  let playerTwo = "⭕️"
-  if(turn === 0 && squares[index] == null){
-    squares[index] = playerOne
-    turn = turn += 1
-  this.setState({lastTurn: playerOne, squares: squares,turn: turn})
-  }else if(turn > 0 && squares[index] == null){
-    squares[index] = playerTwo
-    turn = 0
-  this.setState({lastTurn: playerTwo, squares: squares,turn: turn})
+  let {squares, player} = this.state
+  if(player === 1 && squares[index] == null){
+    squares[index] = "❌"
+    player += 1
+  this.setState({squares: squares, player: player})
+  }else if(player === 2 && squares[index] == null){
+    squares[index] = "⭕️"
+    player = 1
+  this.setState({squares: squares, player: player})
```

Comment:
changed playerOne and playerTwo from variables to be the value included in conditional. changed turn to player and updated conditional values. removed lastTurn state update as it was unused.